### ISSUE TITLE
How to get Field of view of other part of ground

### ISSUE THREAD

**Issue by rohaantahir** [saleh1312/football-players-matching#2]
Hi I am working on your project and wanted to know how can we generate a field of view from other part of ground like you are doing in current project from middle of the ground. If I wanted to cover right or left goal view as well and create field of view using that angle what changes are required then. Kindly please guide me as your orignal tutorial is in arabic.
Thanks I would be highly grateful to you.

**Comment by saleh1312** [comment]
yeah of course you can , just redefine the 4 points that used in prespective transform (from camera )and redefine its mapping points in 2d filed (the 2d image) and you can also define more than 4 points and use ( find homogramphy function) instead of (get prespective transform function )

**Comment by rohaantahir** [comment]
Thanks for reply. are these pixel points? like column number and row number ??
